code/modules/plumbing/plumbers/synthesizer.dm

```diff
@@ -19,10 +19,9 @@
 	var/datum/reagent/reagent_id = null
 	///reagent overlay. its the colored pipe thingies. we track this because overlays.Cut() is bad
 	var/image/r_overlay
-	///The amount of reagent dispensable before requiring a refill from a compressed matter cartridge.
-	var/volume_left = 0
-	///The maximum amount of precursor in a synthesizer.
-	var/max_volume = 1000
+
+//NSV 13 Removed the two vars here for matter cartridge requirements. Need speedy heals
```

Comment:
You've still got this here.

Reply:
Removed 'need speedy heals' at the end of the comment.